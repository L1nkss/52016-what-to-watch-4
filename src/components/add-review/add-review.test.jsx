import AddReview from "@components/add-review/add-review";


describe(`Testing AddReview component`, () => {
  it(`Component should successfully rendered`, () => {
    const component = renderer.create(<AddReview isError={false} isLoading={false} onSubmit={() => {}} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
