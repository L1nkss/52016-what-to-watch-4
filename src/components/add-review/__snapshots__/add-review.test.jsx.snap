// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AddReview component Component should successfully rendered 1`] = `
<div
  className="loading-page"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
